src/Microsoft.AspNetCore.Mvc.Abstractions/Filters/ResultExecutingContext.cs

```diff
@@ -23,10 +23,29 @@ public class ResultExecutingContext : FilterContext
             IList<IFilterMetadata> filters,
             IActionResult result,
             object controller)
+            : this(actionContext, filters, result, controller, ResultExecutionScope.Action)
+        {
+        }
+
+        /// <summary>
+        /// Instantiates a new <see cref="ResultExecutingContext"/> instance.
+        /// </summary>
+        /// <param name="actionContext">The <see cref="ActionContext"/>.</param>
+        /// <param name="filters">All applicable <see cref="IFilterMetadata"/> implementations.</param>
+        /// <param name="result">The <see cref="IActionResult"/> of the action and action filters.</param>
+        /// <param name="controller">The controller instance containing the action.</param>
+        /// <param name="resultExecutionScope"></param>
+        public ResultExecutingContext(
+            ActionContext actionContext,
+            IList<IFilterMetadata> filters,
+            IActionResult result,
+            object controller,
+            ResultExecutionScope resultExecutionScope)
             : base(actionContext, filters)
         {
             Result = result;
             Controller = controller;
+            Scope = resultExecutionScope;
```

Comment:
If the scope is null - does this mean a "global" scope ? Would be great to have this info in the corresponding <param  /> comments.